course7/course_en.md

```diff
@@ -42,14 +42,34 @@ fn init {
 
 Multiple identifiers pointing to the same mutable data structure can be considered aliases, which need to be handled carefully.
 
+```moonbit
+fn alter(a: Ref[Int], b: Ref[Int]) {
+  a.val = 10
+  b.val = 20
+}
+
+fn init {
+  let x: Ref[Int] = { val : 1 }
+  alter(x, x)
+  println(x.val.to_string()) // x.val will be changed twice
+}
+```
+
+## Debugger
+
+- Moon Rabbit's debugger allows us to see real-time running data during operation and better understand the running process
+
+![](../pics/debugger.png)
+
 ## Loops
 
 We can define loops using variables in MoonBit. A loop includes defining the loop variable and its initial value, checking whether to continue the loop, and iterating the variable.
 
 ```moonbit
 let mut i = 0
-while i < 2, i = i + 1 {
+while i < 2 {
   println("Output")
+  i = i + 1
```

Comment:
MoonBit recently added support for the `+=` operator. We can update our sample code to reflect this change.